### PR TITLE
updated version number to v1.5.5

### PR DESCRIPTION
**Pull request by Cyan4973**
also : updated man pages

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,21 @@
+v1.5.5 (Mar 2023)
+fix: fix rare corruption bug affecting the high compression mode (#3517) by @terrelln
```
Comment (ghost) [inline CHANGELOG]: Intentionally finding such corruption bug is great, I would suggest to add discoverer's name/id. It seems he is a testing team member.

Comment (Cyan4973) [reply CHANGELOG]: Discoverer's name is planned to be part of the release note.

But indeed, it could be added to the changelog too.

Comment (gaby) [reply CHANGELOG]: @Cyan4973 Is this getting released today or moved to April?

Comment (Cyan4973) [reply CHANGELOG]: It's coming

Comment (danlark1) [reply CHANGELOG]: Not really a testing team member. Just the owner of ZSTD sync at Google and adviser on APIs, etc :)

**Pull request merged**